src/main/java/com/feywild/feywild/world/gen/OreType.java

```diff
@@ -123,4 +107,23 @@ public PlacedFeature getAlfheimFeature() {
     public HeightProvider getHeight() {
         return UniformHeight.of(VerticalAnchor.absolute(getMinHeight()), VerticalAnchor.absolute(getMaxHeight()));
     }
+
+    public LazyValue<PlacedFeature> setFeature(Block block, RuleTest oreRule, Boolean isRare) {
```

Comment:
Should be `private static`

Reply:
Also a name like `createPlacement` would match the method better.

Reply:
I can make it private but if I make it static it gives alot of errors.

Reply:
My bad, just `private` is fine